doc/src/stdlib/base.md

```diff
@@ -110,6 +110,7 @@ Base.isleaftype
 Base.typejoin
 Base.typeintersect
 Base.Val
+Core.Void
```

Comment:
No, but even if we change to `Base.Void` it will show as `Core.Void` in the rendered docs.